similarity_processor/similarity_cmd.py

```diff
@@ -4,12 +4,12 @@
 import argparse
 sys.path.append(os.path.abspath(os.path.join
                                 (os.path.dirname(__file__), os.pardir
-                                 )))
+                                 )))# pragma: no mutate
```

Comment:
Could you please justify why no mutate pragma was added. I am asking for justification only for record-keeping purposes!